### ISSUE TITLE
first one?

### ISSUE THREAD

**Issue by 1453042287** [qjadud1994/RetinaNet_tensorflow#1]
why the loss is so small? like this:
---------- open MonitoredTrainingSession
---------- run pretrain op
---------- run sync op
STEP : 0	TRAIN_TOTAL_LOSS : 1.21953082	TRAIN_LOC_LOSS : 0.08750351	TRAIN_CLS_LOSS : 1.13203
STEP : 50	TRAIN_TOTAL_LOSS : 1.22364414	TRAIN_LOC_LOSS : 0.09114464	TRAIN_CLS_LOSS : 1.13250
STEP : 100	TRAIN_TOTAL_LOSS : 1.21648896	TRAIN_LOC_LOSS : 0.08427663	TRAIN_CLS_LOSS : 1.13221

**Comment by qjadud1994** [comment]
Since the backbone(Resnet) uses imagenet pretrained weight, learning proceeds steadily.

If you do not use pre-trained weights, your learning will not work.
